server/services/v1/management.go

```diff
@@ -105,10 +108,26 @@ func (m *managementService) CreateNamespace(ctx context.Context, req *api.Create
 		}
 		code = maxId + 1
 	}
+
+	meta := metadata.NewNamespaceMetadata(code, id, req.GetName())
+
+	// Create a Billing account
+	billingId, err := m.BillingProvider.CreateAccount(ctx, id, req.GetName())
```

Comment:
log this error as well, @pboros is there an easy way to emit a metric for this error?